esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -536,7 +601,10 @@ def copy_coords(src_coords, add_method):
 
 
 def _regrid_rectilinear_to_unstructured__prepare(
-    src_grid_cube, target_mesh_cube, precomputed_weights=None
+    src_grid_cube,
+    target_mesh_cube,
+    method="conservative",
```

Comment:
```suggestion
    method,
```

Only called by `GridToMeshESMFRegridder` and `regrid_rectilinear_to_unstructured`, both of which have the default covered - those being the 'public ways in'.